integration/smartbft/smartbft_block_deliverer_test.go

```diff
@@ -205,13 +218,42 @@ var _ = Describe("Smart BFT Block Deliverer", func() {
 		nwo.WaitUntilEqualLedgerHeight(network, channel, 11, network.Peers[0])
 	})
 
-	It("block censorship", func() {
+	FIt("block censorship peer", func() {
```

Comment:
Apparently you can't do programmatic focus in CI, you disabled all other tests...

Reply:
fixed